pages/home/oracle-integrity-staking.mdx

```diff
@@ -0,0 +1,36 @@
+# Oracle Integrity Staking (OIS)
+
+This document outlines the design principles and implementation details of the [Oracle Integrity Staking (OIS)](https://staking.pyth.network/) protocol.
+
+## Design Principles
+
+OIS's economic design focuses on awarding and penalizing stakers over the primary dimension of data accuracy.
+
+Stakers are incentivized to help maintain data quality by receiving rewards from an open-ended pool. However, they also face the risk of having their stake slashed as a penalty for failing to maintain data accuracy.
+
+The core design principles behind OIS include the following:
+
+- Oracle Integrity Staking secures all current and future price feeds produced by the Pyth Network.
+- Data Publishers are individually responsible for data accuracy.
+- Rewards and penalties are proportionate to the stake assigned to each publisher. Delegators share the risks and rewards of the publisher(s) to whom they assign their stake.
+- A higher number of publishers for each price feed contributes positively to the security of such feed.
+- Staking for **OIS** is complementary to staking for **governance**, and eligible $PYTH tokens can be used for both purposes.
+- The ability to slash stake in OIS requires **unlocked** \$PYTH tokens, whereas staking for governance can use both locked and unlocked $PYTH tokens.
+- All parameter related to the OIS protocol are subject to the governance of the Pyth DAO.
+
+## Implementation
+
+OIS implements the design principles above through the following structure:
+
+1. OIS is subject to the same 7-day epoch as governance voting. All parameters used in the OIS protocol are captured at each start of the epoch on Thursdays at 0:00 UTC and remain constant until the end of the epoch. Staking into OIS is also subject to warmup and cooldown period prior and post epoch respectively.
+
+2. Each publisher is programmatically assigned a staking pool where they can self-stake and to which other stakers can delegate.
+   - The staking pool assigned to each publisher covers all price feeds/symbols they publish.
+   - Each staking pool has a soft cap. This soft cap dynamically expands and shrinks given the number of symbols published by the assigned publisher.
+   - Price feeds with a low number of publishers contribute more to the cap's expansion.
+   - Staking into the pool can exceed the soft cap. However no rewards are paid nor penalties are levied on the excess amount.
+   - The OIS protocol prioritizes self-stake attributed to the **publisher's stake** when distributing rewards to the publisher's pool.
+   - All staking pools charge the same delegation fee for stakers who are delegating stake to one or many pools.
+3. Each pool has a maximum reward rate per epoch, which applies only to the staked amount within the soft cap.
```

Comment:
can we make it clear that the cap is a percentage?

Reply:
a hard percentage cap